### ISSUE TITLE
WMSLayer - issue loading more than once [4.4]

### ISSUE THREAD

**Issue by nickcam** [Esri/jsapi-resources#89]
I'm hitting what I think is a bug with WMSLayer. If I try to load more than one the following error occurs in WMSLayer.js and the layer doesn't load:
'Cannot read property 'sublayers' of undefined.

See the below plunk for an example:
[https://plnkr.co/edit/C2VoONjLwueGOh2GGzW7](https://plnkr.co/edit/C2VoONjLwueGOh2GGzW7)

The plunk demonstrates one way of seeing the error, by adding two WMS Layers to the map.
It also occurs if you add one layer, remove it and add it again. I first encountered it by calling load(). If you call wmsLayer.load(), the first time it will work, then the next time it will fail with the same error...even if it hasn't been added to the map.

So basically what I've found is that only the first time load() get's called on any WMSLayer will it work. Any subsequent call to load() will fail on any WMSLayer.

**Comment by bsvensson** [comment]
Yes, this is a bug with the WMSLayer at version 4.4.
Also reported at https://geonet.esri.com/thread/198211-possible-bug-wmslayer-in-44-version-adding-multiple-wmslayers-to-map.  As mentioned there, this is planned to be fixed for version 4.5

**Comment by nickcam** [comment]
Ok cool. Thanks!
